packages/core/botpress/src/web/components/Content/Select/index.js

```diff
@@ -114,15 +114,21 @@ class SelectContent extends Component {
     this.setState({ newItemData: data })
   }
 
-  resetCreateContent = (resetSearch = false) => () => {
+  resetCreateContent = (resetSearch = false) => response => {
+    const item = Object.assign({ id: response.data }, this.state.newItemData)
```

Comment:
Just a matter of my personal taste but I would put the `id` part last to emphasize it should always overwrite whatever is in the `newItemData`